spec/furniture/marketplace/cart_product_component_spec.rb

```diff
@@ -0,0 +1,27 @@
+require "rails_helper"
+
+RSpec.describe Marketplace::CartProductComponent, type: :component do
+  subject(:output) { render_inline(component) }
+
+  let(:operator) { create(:person, operator: true) }
+
+  let(:cart) { create(:marketplace_cart) }
+  let(:marketplace) { cart.marketplace }
+  let(:product) { create(:marketplace_product, :with_description, :with_photo) }
+  let(:cart_product) { create(:marketplace_cart_product, cart: cart, product: product) }
+
+  let(:component) { described_class.new(cart_product: cart_product, current_person: operator) }
+
+  it { is_expected.to have_content(product.name) }
+  it { is_expected.to have_content(product.description) }
+  it { is_expected.to have_content(helpers.humanized_money_with_symbol(product.price)) }
+  it { is_expected.to have_link(I18n.t("marketplace.cart_product_component.add")) }
+  it { is_expected.to have_link(I18n.t("marketplace.cart_product_component.remove")) }
+  xit { is_expected.to have_selector("img[src$='#{product.photo.filename}']") }
```

Comment:
This fails even with vips installed; but it passes locally...

I'm going to pull it (and the vips install stuff) into their own PR so the rest of the work can flow forward.

Reply:
Couldn't pull the vips install because it is necessary to render the component.